### PR TITLE
[fix] rename the pub commands to include dart

### PR DESCRIPTION
**Pull request by suthar26**

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -18,9 +18,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install dependencies
-        run: pub get
+        run: dart pub get
```
Comment (kaushalkapasi) [inline .github/workflows/publish.yaml:21]: should all be `flutter` instead of `dart`

Comment (kaushalkapasi) [reply .github/workflows/publish.yaml:21]: https://docs.flutter.dev/development/packages-and-plugins/developing-packages#publish

Comment (suthar26) [reply .github/workflows/publish.yaml:21]: dart will work the same way, I believe they work the same way, believe it would be getting proxied

Comment (kaushalkapasi) [reply .github/workflows/publish.yaml:21]: flutter commands run the respective native build / install commands which dart by itself won't...

Comment (suthar26) [reply .github/workflows/publish.yaml:21]: also would be better to use the actual docker image compared to an action built and managed by an individual
https://github.com/subosito/flutter-action

Comment (kaushalkapasi) [reply .github/workflows/publish.yaml:21]: yeah, want to base it the other workflow for Unit Testing? it leverages that

Comment (suthar26) [reply .github/workflows/publish.yaml]: fixed :)

File: .github/workflows/publish.yaml [hunk 2]
```diff
@@ -29,6 +32,6 @@ jobs:
         run: |
           sh ./pub_login.sh
       - name: Check Publish Warnings
-        run: pub publish --dry-run
+        run: dart pub publish --dry-run
```
Comment (kaushalkapasi) [inline .github/workflows/publish.yaml:35]: mix of dart and flutter

File: .github/workflows/publish.yaml [hunk 3]
```diff
@@ -9,18 +9,21 @@ on:
 
 jobs:
   build:
-    name: Release Flutter SDK
```
Comment (kaushalkapasi) [inline .github/workflows/publish.yaml:12]: deleted `name`?

**Review by JamieSinn — APPROVED** [review]

**Pull request merged**